lib/run/upload-run.js

```diff
@@ -0,0 +1,257 @@
+const _ = require('lodash'),
+    sdk = require('postman-collection'),
+    util = require('../util'),
+    print = require('../print'),
+    retry = require('async-retry'),
+    request = require('postman-request'),
+
+    POSTMAN_UPLOAD_ERROR_LINK = '<TBA>',
+    UPLOAD_RUN_API_URL = '<TBA>',
+    NEWMAN_STRING = 'newman';
+
+/**
+ * @private
+ * Internal upload call
+ *
+ * @param {Object} uploadOptions
+ * @returns {function} Returns an async function which can be used by async-retry library to have retries
+ */
+_upload = (uploadOptions) => {
```

Comment:
If this is going to return a function, it should be called something like `_getUploadRetryable`.
Or, don't return a function! Just bind the params where you're using it.